Casks/vertcoin-core.rb

```diff
@@ -0,0 +1,19 @@
+cask "vertcoin-core" do
+  version "0.18.0"
+  sha256 "2c040ca687297ec8124251833e7c2a5a9d3831b1eaacab681cc8772b353e6702"
+
+  url "https://github.com/vertcoin-project/vertcoin-core/releases/download/#{version}/vertcoin-#{version}-osx.dmg",
+      verified: "github.com/vertcoin-project/vertcoin-core/"
+  name "Vertcoin Core"
+  desc "Vertcoin client and wallet"
+  homepage "https://vertcoin.org/"
+
```

Comment:
```suggestion

  livecheck do
    url :url
    regex(/href=.*?vertcoin[._-]v?(\d+(?:\.\d+)+)(?:[._-]osx)?\.dmg/i)
    strategy :github_latest
  end
```

It's technically possible to check the Git tags using the standard regex for tags like 1.2.3/v1.2.3 (`/^v?(\d+(?:\.\d+)+)$/i`) but this cask uses a release download, so we need to check releases instead. For now, this can simply use the `GithubLatest` strategy (with a regex that matches the version from the dmg file).

If this fails in the future (e.g., a release doesn't include the dmg file), we can switch to checking the releases page (while omitting prerelease versions using `?q=prerelease%3Afalse` in the URL).